recipes/sherlockml-boltzmannclean/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sherlockml-boltzmannclean" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "0c028e9f8a67fac787587c20e24ef69792899d1c76e1313515518a9aff0f57ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
It looks like this recipe might be a candidate for `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages